src/main/java/com/alexfu/sqlitequerybuilder/builder/CreateTableSegmentBuilder.java

```diff
@@ -26,17 +27,14 @@ public CreateTableSegmentBuilder ifNotExists() {
   }
 
   public CreateTableSegmentBuilder column(Column column) {
-    if (column == null) {
-      throw new IllegalArgumentException("A non-null column is required.");
-    }
+    Preconditions.checkNotNull(column, "A non-null column is required.");
     definitions.add(column);
     return this;
   }
 
   public CreateTableSegmentBuilder table(String name) {
-    if (name == null || name.isEmpty()) {
-      throw new IllegalArgumentException("Table name can not be empty.");
-    }
+    Preconditions.checkNotNull(name, "Table name can not be empty.");
+    Preconditions.checkNotEmpty(name, "Table name can not be empty.");
```

Comment:
With these 2 lines checkNotNull will be called twice. Once here and another inside checkNotEmpty implementation.